src/Configuration/ValueObject/ValueObject.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorConfig\Configuration\ValueObject;
+
+interface ValueObject
```

Comment:
k čemu je tenhle interface?

Reply:
Je to len znackovaci interface, .... pouziva sa najma v testoch, aby bolo jasne ze co sa tam ocakava za parametre vystupy.

Reply:
To mi přijde nadbytečné - kvuli testům mít interface. Prostě v testu nebude typehint, ne?

Reply:
Kludne to mozem zmazat, mna tam jeden interface naviac pre lahsie pochopenie kodu nejak nevyrusuje.